docs/guides/migrating-from-camunda-platform-7/index.md

```diff
@@ -0,0 +1,56 @@
+---
+id: index
+title: Before starting a migration
```

Comment:
"Before starting a migration" may not be the cleanest title we could provide here -- perhaps just "Migration details" or "Overview"?

Reply:
I've gone back and forth on this, avoiding "Overview" and "Introduction" since they lack context. I want people to go to this page first. With that context, does that change your recommendation?

Reply:
Perhaps "Pre-migration details" or "Preparing for migration"?

Reply:
Ooo! Let's try "Pre-migration details".